src/Controls/src/Core/VisualElement.cs

```diff
@@ -701,7 +701,7 @@ public bool IsPlatformEnabled
 
 				_isPlatformEnabled = value;
 				if (value && IsPlatformStateConsistent)
-					InvalidateMeasureInternal(InvalidationTrigger.RendererReady);
+					//InvalidateMeasureInternal(InvalidationTrigger.RendererReady);
```

Comment:
Is this intentional? It seems that the flow is now different with the if block controlling the InvalidateStateTriggers when it would hav always run?